llvm/lib/ProfileData/SampleProfReader.cpp

```diff
@@ -902,12 +890,16 @@ std::error_code SampleProfileReaderExtBinaryBase::readFuncProfiles() {
       for (const auto &NameOffset : FuncOffsetList) {
         const auto &FContext = NameOffset.first;
         auto FName = FContext.getName();
+        StringRef FNameString;
```

Comment:
auto FName = FContext.GetName() --> it is hard to tell the type of Fname by looking at it. Either using explicit type, or change the getter method name.